TYPO3.Flow/Classes/TYPO3/Flow/Object/Configuration/ConfigurationBuilder.php

```diff
@@ -80,10 +80,6 @@ public function buildObjectConfigurations(array $availableClassAndInterfaceNames
                     continue;
                 }
 
-                if ($this->reflectionService->isClassFinal($classOrInterfaceName)) {
-                    continue;
-                }
-
```

Comment:
Without this code, we create object configuration for all final classes, but I don't think that this would cause any harm